pyquil/tests/test_qpu.py

```diff
@@ -1,268 +0,0 @@
-import networkx as nx
```

Comment:
Essentially all these tests were being skipped in CI. We can follow-up in a later PR to add `QPU` test coverage